docs/Walkthrough.md

```diff
@@ -1,88 +1,181 @@
 # BTCPay Server Walkthrough
 
-In this article, we will walk you through the **BTCPay user interface** and show you how to navigate the different options.
+This article will walk you through the **BTCPay user interface** and show you how to navigate the different options.
 
-Watch video below for an interactive overview of the features.
+Watch the video below for an interactive overview of the features.
 
 [![BTCPay Server Walkthrough](https://img.youtube.com/vi/ZIfJyq9RimM/mqdefault.jpg)](https://www.youtube.com/watch?v=ZIfJyq9RimM "BTCPay Server - WalkThrough")
 
-After you've created an account on the BTCPay Server instance hosted by yourself or a third-party, you'll see a variety of options in the top navigation menu.
+After you've created an account on the BTCPay Server instance hosted by yourself or a third party, you'll be greeted by your store's new home or, as we like to call it, your Dashboard.
 
-![BTCPay Server Navigation](./img/BTCPay-Navigation.jpg "BTCPay Server Navigation")
+![BTCPay Server Navigation](./img/walktrough/welcome-store.jpg "BTCPay Server Navigation")
 
-* Server settings
-* Stores
-* Apps
-* Wallets
-* Invoices
-* Payment requests
-* My settings
-* Notifications
+All the settings in the left menu are meant for the Current Store you selected at the top. 
 
-## Server settings
-
-**Server settings** is something only a server admin can access. If you're using someone else's server, you won't see Server Settings. Inside the settings, you can perform tasks such as controlling users, rates, updating the server, etc. For more information, check [Server Settings FAQ](./FAQ/ServerSettings.md)
+![BTCPay Server Navigation](./img/walktrough/selected-store.jpg "BTCPay Server Navigation")
 
-- Users - Add, remove or manage users of your BTCPay Server.
-- Email server - If you want users to verify their e-mail address when registering set up the SMTP settings.
-- Policies - Enable or disable user registration, e-mail verification, search engine indexing, display apps on website root.
-- Services - gRPC, REST and RTL used for connecting your LN node, SSH keys and uploaded file storage setup.
-- Theme - Customize the front-end appearance of your BTCPay Server.
-- Maintenance - Update your BTCPay to latest version and clean your BTCPay by deleting unused docker images.
-- Logs - Displays most recent logs of the BTCPay Server.
-- Files - After enabling this feature in Services, upload external files and access them via url.
+* [Notifications](Walkthrough.md#notifications)
+* Dashboard 
+* [Settings](Walkthrough.md#store) **store settings** 
+* [Wallets](Walkthrough.md#wallets)
+    * Bitcoin
+    * Lightning
+* [Payments](Walkthrough.md#payments)
+    * Invoices
```

Comment:
Maybe we can link to this pages since we have docs for most, if not all of them?